### PR TITLE
feat: Normalize in rust flag

### PR DESCRIPTION
**Pull request by mitsuhiko**

### CODE REVIEW

File: src/sentry/options/defaults.py [hunk 1]
```diff
@@ -157,3 +157,4 @@
 # Ingest refactor
 register('store.process-in-kafka', type=Bool, default=False)
 register('store.kafka-sample-rate', default=0.0)
+register('store.normalize-in-rust', thype=Sequence, default=[])
```
Comment (tonyo) [inline src/sentry/options/defaults.py]:
```suggestion
register('store.normalize-in-rust', type=Sequence, default=[])
```

**Review by tonyo — COMMENTED** [review]
Did a quick check in `sentry shell`, getting/setting list values seem to work fine.
We shouldn't forget to set the initial value explicitly during the deploy, though, otherwise we'll be hitting the DB for every query, unfortunately.

**Review by untitaker — APPROVED** [review]
I am reading `defaults.py` for the first time

**Review by untitaker — APPROVED** [review]
great work

**Pull request merged**